low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -169,7 +163,7 @@ class SSPLDevDeploy:
             _ = self._execute_cmd(cmd)
 
             # cortx-py-utils dependency
-            cmd = "python3 -m pip -r https://raw.githubusercontent.com/Seagate/cortx-utils/main/py-utils/requirements.txt"
+            cmd = "python3 -m pip install -r https://raw.githubusercontent.com/Seagate/cortx-utils/main/py-utils/requirements.txt"
```

Comment:
so cortx-py-utils is not installing its dependencies ?

Reply:
```Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : cortx-py-utils-1.0.0-102_23eb199.noarch                                                                              1/1
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3.6 install --user` instead.
Collecting aiohttp==3.6.1 (from -r /opt/seagate/cortx/utils/conf/requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/e0/cb/5fa1d7722488995358d6092a7d2ccceb9011ab78842ccd336fbb5ec0d878/aiohttp-3.6.1-cp36-cp36m-manylinux1_x86_64.whl```

I observed that it gets installed with `cortx-py-utils`.

Reply:
It wasn't earlier. Now we can see its dependencies also getting installed.
I will remove this line.

Reply:
Done.